### ISSUE TITLE
Save a map as simple web application (a group of files, based on leaflet.js)

### ISSUE THREAD

**Issue by mkgrgis** [dbeaver/dbeaver#12245]
[xf:gis]
**Is your feature request related to a problem? Please describe.**
For sending a small geo datasets to other people it will be useful to have this feature, based on files from `/tmp/.dbeaver... `
![изображение](https://user-images.githubusercontent.com/41448637/116032842-0d4f2680-a669-11eb-9150-92ef7bc9be6a.png)

**Describe the solution you'd like**
A new button - save as simple web app. Maybe as zip from 
```
├── inc
│   ├── layers.png
│   ├── leaflet.css
│   ├── leaflet.js
│   ├── wkx.js
│   └── wkx.min.js
└── *.html
```
If Dbeaver team will refactor web geo block (leaflet map) to geoJSON (see https://github.com/dbeaver/dbeaver/issues/12244), no need to load wkx.* JavaScript. 

**Describe alternatives you've considered**
No alternatives, most of functions yet realised.

**Comment by kseniiaguzeeva** [comment]
We have _Copy as picture/Save as picture_ options. Are they inconvinient?
![116032842-0d4f2680-a669-11eb-9150-92ef7bc9be6a](https://user-images.githubusercontent.com/51405061/118497394-b3082980-b72d-11eb-8616-1f09051bd552.png)

**Comment by mkgrgis** [comment]
No problems, @kseniiaguzeeva , but i want to send to colleges not a picture, but a simple live web app. This web app works in case of "Open in browser" calling. This web app allow to zoom in, zoom out, switch some layers, press on point and read a data.

**Comment by kseniiaguzeeva** [comment]
**Open in Browser** already does it. I see your FR #12244. Am I correct that this ticket is continuation of 12244?

**Comment by mkgrgis** [comment]
No, this ticket isn't a continuation of https://github.com/dbeaver/dbeaver/issues/12244. Yes, this two tickets is about map subsystem, but programmatically it's independent.
This ticket is about saving a web application as archive or a group of files, about sending HTML/JavaScript _live web map app_ as group of files to other users.

**Comment by kseniiaguzeeva** [comment]
Please open map view in browser and copy-paste the way to File explorer.  You will find the files there.
![W8usTFNXTv](https://user-images.githubusercontent.com/51405061/119101091-4b015e00-ba21-11eb-8d2e-9025f4d2ed49.jpg)

**Comment by mkgrgis** [comment]
Thanks, this works. Maybe there is reason to make special button "Open a folder of web application for this map" ? It's about my this FR begins https://github.com/dbeaver/dbeaver/issues/12245#issue-867320716.

**Comment by kseniiaguzeeva** [comment]
I'm afraid, the case is quite rare and the Gis toolbar will be overloaded by one more button.  Also the simple workaround exists. As a solution I suggest to leave the ticket open to look whether other people will vote for this feature.

**Comment by mkgrgis** [comment]
Thanks, @kseniiaguzeeva !
